ssh2/ssh2.php

```diff
@@ -177,6 +177,18 @@
  */
 function ssh2_connect ($host, $port = null, array $methods = null , array $callbacks = null ) {}
 
+/**
+ * (PECL ssh2 &gt;= 1.0)<br/>
+ * Close a connection to a remote SSH server
+ * @link http://php.net/manual/en/function.ssh2-disconnect.php
+ * @param resource $session <p>
+ * An SSH connection link identifier, obtained from a call to
+ * ssh2_connect.
+ * </p>
+ * return bool
```

Comment:
`@return bool`

Reply:
mh, the PR is missing the latest commit ...

Reply:
That's strange. Commit view gives 404 too: https://github.com/JetBrains/phpstorm-stubs/commit/00b3da8f2f2090a5edffc4f89d5a4ee79b6c5a21

Reply:
Link opens now, no 404 anymore. But commit is still not present in PR. Could you please try to rebase/force push branch or recreate PR?